libs/ledger-live-common/src/ble/hooks/useBleDevicePairing.ts

```diff
@@ -31,8 +32,19 @@ export const useBleDevicePairing = ({
   const [pairingError, setPairingError] = useState<PairingError>(null);
 
   useEffect(() => {
-    const requestObservable = withDevice(deviceId)(t => from(getVersion(t))).pipe(first());
+    const requestObservable = withDevice(deviceId)(t => from(getVersion(t))).pipe(
+      first(),
+      retry({
+        delay: (err: BleError) => {
+          if (err instanceof LockedDeviceError) {
```

Comment:
Can you just add a small line about the fact that on `LockedDeviceError` it automatically retries (but still propagated the error) in the doc comments above the function definition pls ?